### DISCUSSION TITLE
minor bugs in the demo

### DISCUSSION THREAD

**Discussion by vorosj** [rzeldent/esp32-smartdisplay#112] [Q&A] [answered]
The light value and button count value is not visible. 
Reason: the ui_lblCountValue and ui_lblCdrValue label points outside the visible area.
I deleted the lv_obj_set_align(xxx, LV_ALIGN_TOP_MID) lines, and added some x offset, and works

**Comment by rzeldent** [comment, marked as answer]
Hi,

In the develop I made the image smaller so it does no longer cover the labels...
